README.md

```diff
@@ -50,38 +50,31 @@ This includes:
 - when there are aliases, listing the preferred spelling first
 - adding the statement `"version" : "None"`
 
-## How to install the headers
+### CMake
 
-```
-mkdir build
-cd build
-cmake ..
-cmake --build . --target install
-```
+Regardless of whether you consume SPIRV-Headers via `find_package` or `add_subdirectory` after those calls succeed then you will `link` against the header only library.
+`Linking` against a header only library is just a nice way to add the include directory without having to know the project structure, which can be subject to change.
 
-Then, for example, you will have `/usr/local/include/spirv/unified1/spirv.h`
+```cmake
+# Example usage of add_subdirectory
+add_executable(foobar)
+add_subdirectory(SPIRV-Headers)
 
-If you want to install them somewhere else, then use
-`-DCMAKE_INSTALL_PREFIX=/other/path` on the first `cmake` command.
-
-## Using the headers without installing
+target_link_libraries(foobar PRIVATE SPIRV-Headers::SPIRV-Headers)
+```
 
-### Using CMake
-A CMake-based project can use the headers without installing, as follows:
+```cmake
+# Example usage of find_package
+add_executable(foobar)
+find_package(SPIRV-Headers REQUIRED CONFIG)
 
-1. Add an `add_subdirectory` directive to include this source tree.
-2. Use `${SPIRV-Headers_SOURCE_DIR}/include}` in a `target_include_directories`
-   directive.
```

Comment:
NOTE: This is bad practice we shouldn't encourage projects to do. Since it relies on implementation details of `SPIRV-Headers` which shouldn't be relied on.

It's much better to link against the header only library instead.